utils/pom.xml

```diff
@@ -72,6 +77,10 @@
       <artifactId>bcprov-jdk15on</artifactId>
     </dependency>
     <dependency>
+      <groupId>org.bouncycastle</groupId>
```

Comment:
We want dependency explicitly stated, as done for bcprov-jdk15on. The root pom has it as well.